Nuget/Ploeh.AutoFixture.AutoFakeItEasy.nuspec

```diff
@@ -8,7 +8,7 @@
         <owners>Nikos Baxevanis</owners>
         <dependencies>
             <dependency id="AutoFixture" version="@build.number@" />
-            <dependency id="FakeItEasy" version="[1.7,2)" />
+            <dependency id="FakeItEasy" version="1.7" />

```

Comment:
I am not sure. I made the change based on the thought that it was an error to have introduced it and your comment https://github.com/AutoFixture/AutoFixture/issues/663#issuecomment-230164610, which seemed to be in favour of the change.

My understanding is that new projects would still depend on 1.7. It's only if someone runs a `nuget update` or equivalent that there'd be a change, yes? You concern is that all of a sudden clients would have an undesirable FakeItEasy, which will be incompatible with their code? I guess that's the case. I can restore the original.